charts/policy-controller/values.yaml

```diff
@@ -37,6 +37,10 @@ webhook:
     capabilities:
       drop:
         - ALL
+  podDisruptionBudget:
+    enabled: true
+    minAvailable: 1
+    # maxUnavailable: 3
```

Comment:
Nit: Is this left here on purpose?

Reply:
If this value is not set here it will be empty and unexpressed in the actual resource.
This is a pattern that's used in a lot of places, though happy to consider alternatives!

Reply:
Yeah, I'm not an helm expert, and saw it commented out, and wasn't sure if it was just left over from developing, so figured I'd ask :)